src/group-constructors.js

```diff
@@ -55,6 +55,8 @@
     var underlyingPlayer;
     var ticker = function(tf) {
       var player = underlyingPlayer._wrapper;
+      if (player.playState == 'pending') return;
```

Comment:
I'm not sure. Seems like there could be cases where we never get another call now, but I didn't look in detail.